src/Slices/ServiceInventory/UI/Tabs/Tabs.tsx

```diff
@@ -235,17 +235,17 @@ const documentationTab = (
 const getAttributeValue = (attributeName: string, row: Row) => {
   if (
     row.attributes &&
-    row.attributes.active &&
```

Comment:
I just want to know why the order is reversed? Not saying that is wrong but wonder, should candidate be default displayed value here?

Reply:
Bart requested it :D